src/dispatch/static/dispatch/src/api/menu.js

```diff
@@ -110,18 +110,25 @@ const Menu = [
     href: "/plugins"
   },
   {
-    title: "Workflows",
-    group: "workflows",
-    component: "Workflows",
-    icon: "work",
-    href: "/workflows"
+    title: "Tag Types",
+    group: "Configuration",
+    name: "Tag Types",
+    icon: "settings",
```

Comment:
Shouldn't we pick a different icon?

Reply:
I'll change it to label, so we're consistent with the other tag section above.